Gopkg.toml

```diff
@@ -14,7 +14,7 @@
 
 [[constraint]]
   name = "github.com/getkin/kin-openapi"
-  version = "0.1.0"
+  branch = "master"
```

Comment:
I suppose they stopped tagging their merges?

Reply:
That's what it looks like.